backend/src/main/java/corea/exception/ExceptionType.java

```diff
@@ -10,7 +10,9 @@ public enum ExceptionType {
     NOT_FOUND_ERROR(HttpStatus.NOT_FOUND, "해당하는 값이 없습니다."),
     AUTHORIZATION_ERROR(HttpStatus.UNAUTHORIZED, "인증에 실패했습니다."),
     ALREADY_APPLY(HttpStatus.BAD_REQUEST, "해당 방에 이미 참여했습니다"),
-    PARTICIPANT_SIZE_LACK(HttpStatus.BAD_REQUEST, "참여 인원 수가 부족합니다.");
+    PARTICIPANT_SIZE_LACK(HttpStatus.BAD_REQUEST, "참여 인원 수가 부족합니다."),
+    NOT_MATCHED_MEMBER(HttpStatus.BAD_REQUEST, "매칭된 인원들이 아닙니다."),
+    ALREADY_REVIEW_COMPLETED(HttpStatus.BAD_REQUEST, "이미 리뷰를 완료했습니다.");
```

Comment:
[제안] 

나중에 에러 타입 네이밍 컨벤션 맞추는 것도 좋을 지도요?

```suggestion
    NOT_MATCHED_MEMBER(HttpStatus.BAD_REQUEST, "매칭된 인원들이 아닙니다."),
    ALREADY_COMPLETED_REVIEW(HttpStatus.BAD_REQUEST, "이미 리뷰를 완료했습니다.");
```